gui/mouse.tcl

```diff
@@ -917,6 +952,94 @@ proc button3node { c x y } {
 	    .button3menu add command -label "Mail client" \
 		-state disabled
 	}
+	###----------------------------------------------------------------------
```

Comment:
I don't think apache, bind, dhcpd should be here in this menu: that's why we have the "Services" menu. Also, does this work in Linux or only in FreeBSD?
Additionally, I get an error message "missing close-bracket" when I right-click on a node in exec mode.